<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10894" started-at="2016-05-18T00:59:24Z" finished-at="2016-05-18T00:59:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10894" started-at="2016-05-18T00:59:24Z" finished-at="2016-05-18T00:59:35Z">
      <class name="br.com.unisul.test.TestFormularioDeCadastro">
        <test-method status="PASS" signature="preConditions()[pri:0, instance:br.com.unisul.test.TestFormularioDeCadastro@2a5ca609]" name="preConditions" is-config="true" duration-ms="10045" started-at="2016-05-17T21:59:24Z" finished-at="2016-05-17T21:59:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method status="FAIL" signature="cadastrarUmUsuarioComSucesso()[pri:0, instance:br.com.unisul.test.TestFormularioDeCadastro@2a5ca609]" name="cadastrarUmUsuarioComSucesso" duration-ms="826" started-at="2016-05-17T21:59:34Z" finished-at="2016-05-17T21:59:35Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cadastro realizado com sucesso.] but found [Ocorreu um erro no servidor. Tente novamente mais tarde!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cadastro realizado com sucesso.] but found [Ocorreu um erro no servidor. Tente novamente mais tarde!]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at br.com.unisul.test.TestFormularioDeCadastro.cadastrarUmUsuarioComSucesso(TestFormularioDeCadastro.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cadastrarUmUsuarioComSucesso -->
        <test-method status="PASS" signature="closeDriver()[pri:0, instance:br.com.unisul.test.TestFormularioDeCadastro@2a5ca609]" name="closeDriver" is-config="true" duration-ms="8" started-at="2016-05-17T21:59:35Z" finished-at="2016-05-17T21:59:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
      </class> <!-- br.com.unisul.test.TestFormularioDeCadastro -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
